pypowsybl/network/impl/network.py

```diff
@@ -3196,10 +3198,38 @@ def update_branches(self, df: DataFrame = None, **kwargs: ArrayLike) -> None:
 
             .. code-block:: python
 
-                network.update_branches(element_id='BRANCH_ID', connected_1=False, connected_2=False)
+                network.update_branches(element_id='BRANCH_ID', connected1=False, connected2=False)
```

Comment:
it will be create a breaking change wont it ? maybe we should maintain the old version

Reply:
No, the doc was just wrong